CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 Preferably use **Added**, **Changed**, **Removed** and **Fixed** topics in each release or unreleased log for a better organization.
 
+## [1.0.3](https://github.com/quintoandar/hive-metastore-client/releases/tag/1.0.3)
+### Changed
+* Handling exception when adding duplicate partitions ([#37](https://github.com/quintoandar/hive-metastore-client/pull/37))
+* Remove method returning for create_database_if_not_exists ([#39](https://github.com/quintoandar/hive-metastore-client/pull/39))
```

Comment:
Isn't better to use `Removed`?

Reply:
You mean this?
```suggestion
* Removed method returning for create_database_if_not_exists ([#39](https://github.com/quintoandar/hive-metastore-client/pull/39))
```
Or using the ### Remove?

Reply:
Use the first one